apps/confessions-client/pages/popup.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+
+/**
+ * This page is necessary to receive PCDs from the passport after requesting
+ * a PCD from the passport. It uses the window messaging API to communicate
+ * the PCD it received back to the requesting tab.
+ */
+export default function AuthPopup() {
```

Comment:
this code is now duplicated in a bunch of 3rd party apps, so we should extract it. not relevant for this pr just noting it. here's an issue: https://github.com/proofcarryingdata/zupass/issues/131